pkg/utils/sysinfo.go

```diff
@@ -0,0 +1,16 @@
+package utils
+
+import (
+	"github.com/pbnjay/memory"
+	"runtime"
+)
+
+// GetSysCPUCores returns number of CPU cores on the system
+func GetSysCPUCores() int {
+	return runtime.NumCPU()
+}
+
+// GetSysMemoryMB returns the capacity (in MB) of the physical memory installed on the system
+func GetSysMemoryMB() uint {
+	return uint(memory.TotalMemory() / 1e6)
```

Comment:
Maybe use `2e20` here to get binary MB?  (And call it GetSysMemoryMiB if you really want to be correct per SI.)

Reply:
Or I guess this would be `1<<20` not `2e20`.  (Or just `1048576`, it's not that many more characters.)